components/icon-buttons/Web/Web-icon-buttons.md

```diff
@@ -8,227 +8,167 @@ path: /components/buttons
+const iconToggle = new MDCIconButtonToggle(document.querySelector('.mdc-icon-button'));
+```
 
-		const iconToggle1 = new MDCIconButtonToggle(document.querySelector('.btn2'));
-		iconToggle1.unbounded = true;
-	```
-* In your `*.html` file, you can now add the iterations of the following, which shows containers with background images:
-    ```html
-    <div class="container">
-          <div class="image-with-fave">
-            <img src="https://cdn.glitch.com/4f0b2993-dd70-46bf-916f-a6e81af58957%2Fimage1.jpeg?v=1570131499813" />
-            <button
-              id="bookmark-0"
-              class="mdc-icon-button btn1 fave-button"
-              aria-label="Add to favorites"
-              aria-hidden="true"
-              aria-pressed="false"
-            >
-              <i
-                class="material-icons mdc-icon-button__icon mdc-icon-button__icon--on"
-                >favorite</i
-              >
-              <i class="material-icons mdc-icon-button__icon">favorite_border</i>
-            </button>
-          </div>
-          <div class="image-with-fave">
-            <img
-          src="https://cdn.glitch.com/4f0b2993-dd70-46bf-916f-a6e81af58957%2Fimage2.jpeg?v=1570131499876" />
-            <button
-              id="bookmark-1"
-              class="mdc-icon-button btn2 fave-button"
-              aria-label="Add to favorites"
-              aria-hidden="true"
-              aria-pressed="false"
-            >
-              <i
-                class="material-icons mdc-icon-button__icon mdc-icon-button__icon--on"
-                >favorite</i>
-              <i class="material-icons mdc-icon-button__icon">favorite_border</i>
-              </button>
-          </div>
-          ...
-        </div>
-    ```
+### Icon Button Toggle with SVG
 
-#### Anatomy and key properties
+The icon button toggle can be used with SVGs.
 
-The icon button consists of two icons for 'on' and 'off'.
+```html
+<button id="star-this-item"
+   class="mdc-icon-button mdc-icon-button--on"
+   aria-label="Unstar this item"
+   aria-pressed="true">
+   <svg class="mdc-icon-button__icon">
+     ...
+   </svg>
+   <svg class="mdc-icon-button__icon mdc-icon-button__icon--on">
+     ...
+  </svg>
+</button>
+```
 
-![Toggle button anatomy](assets/toggle_button_anatomy.png)
+### Icon Button Toggle with an Image
 
-1. Container
-1. Icon
+The icon button toggle can be used with `img` tags.
 
-<b>Container attributes</b>
-<br>
+```html
+<button id="star-this-item"
+   class="mdc-icon-button mdc-icon-button--on"
+   aria-label="Unstar this item"
+   aria-pressed="true">
+   <img src="" class="mdc-icon-button__icon"/>
+   <img src="" class="mdc-icon-button__icon mdc-icon-button__icon--on"/>
+</button>
+```
 
-Design Attribute | Theme value | Equivalent Sass mixin attribute
----|---|---
-Container ink color | Primary color | `mdc-button-ink-color($color)`
+## API
 
-<b>Icon attributes</b>
+### CSS Classes
```

Comment:
Titles need to follow sentence case: capitalize the first word, lower-case all following words unless they're acronyms or proper nouns.